source/meetings.rst

```diff
@@ -0,0 +1,53 @@
+.. _meetings:
+
+##################
+Project Meetings 
+##################
+
+All meetings associated with the yt project will be announced publicly to the
+community through communication channels, including slack, the yt-dev and yt-users 
+mailing lists. Meeting notes will be subsequently posted and announced in the
+aforementioned channels and in the `project records 
+<https://github.com/yt-project/project-records>`_ repository. 
+All are welcome to attend, no matter their
+level of participation within the project. If part of the meeting requires a
+closed session, the meeting will adjourn to a closed meeting and notes will be
+made available at a later time. In all cases, meetings will be open for as long
+as possible.
+
+Steering Committee Meetings
+---------------------------
+
+Public meetings, optimally including all members of the steering committee,
+should happen at least once a quarter. These meetings are to encourage frank
+and open discussion about the future of the project. Meetings will happen over
+video chat to encourage remote participation, and times should be chosen to
+accomodate international attendees. The meetings invite will be public, and any
+interested developer or user is welcome to attend. When necessary, steering
+committee meetings may need to adjourn to a closed session for sensitive
+discussions. The steering committee email will not be public to ensure that
+sensitivie discussions may be held on that list. However, all non-sensitive
+items will be reported and documented at subsequent public steering 
+committee meetings. 
+
+Maintainer Coworking Meetings
+-----------------------------
+
+Twice monthly, or as required, video chats should be held to ensure timely review
+of pending maintainence of the project. This may include triaging new issues,
+reviewing new PRs, or submitting bugfixes together. These meetings will attempt
+to address all unaddressed maintenance issues, and any outstanding tasks will
+be discussed and mentioned publicly as a comment on the pull request. Any
+developer, reviewer, maintainer, or user is welcome to attend this meeting.
+Developers who have open pull requests they would like to see reviewed are
+particularly encouraged to attend to aid and facilitate 
+discussion about the pull request.
+Finally, new contributors to the project are especially encouraged to attend. 
+
+Team Meetings
```

Comment:
I think this part can be left out. Any group of people is welcome to meet to discuss specific project issues, but I don't think it serves any purpose to mention it as a thing.

Reply:
Removed!